app/src/main/java/com/github/onlynotesswent/ui/overview/AddNote.kt

```diff
@@ -116,21 +116,17 @@ fun AddNoteScreen(
             verticalArrangement = Arrangement.spacedBy(8.dp)) {
               Spacer(modifier = Modifier.height(30.dp))
 
-              OutlinedTextField(
+              NoteDataTextField(
```

Comment:
Good addition, we will have to then put this common composable in a separate class